expression/builtin.go

```diff
@@ -927,7 +927,7 @@ func (b *baseBuiltinFunc) setDecimalAndFlenForDatetime(fsp int) {
 	b.tp.Decimal = fsp
 	b.tp.Flen = mysql.MaxDatetimeWidthNoFsp + fsp
 	if fsp > 0 {
-		// For `.`
+		// Add the length for `.`.
```

Comment:
```suggestion
		// Add the length for '.' in time like '2000:01:01 00:00:00.000001'.
```

Reply:
I think it's quite understandable. Since the logic is very simple here.

Reply:
There are three comments `Add the length for '.'`